governance/constitution/kms_actions_maa.js

```diff
@@ -173,7 +197,7 @@ actions.set(
             item.forEach((i) => {
```

Comment:
in the non-flattened model we should test if the key represents an object and than drill down in the object to check presence in CLAIMS. Way harder than using flattened claims.